Loop_frontend/app/ideaboard/actions.ts

```diff
@@ -0,0 +1,26 @@
+'use server'
+
+const API_BASE_URL = process.env.NEXT_PUBLIC_API_BASE_URL;
+
+
+// Create a new project
+export async function createProject(projectData: any) {
```

Comment:
`createProject` API route already exists. Migrate the method in the page of the ideaboard here.